kale/publisher.py

```diff
@@ -57,7 +57,7 @@ def publish(self, task_class, task_id, payload,
 
         sqs_queue.send_message(
             MessageBody=kale_msg.encode(),
-            DelaySeconds=delay_sec or 1
+            DelaySeconds=delay_sec or 0
```

Comment:
A more appropriate backwards compatible change would be to change the default value in the parameter list to `delay_sec=1` and remove the `delay_sec is not None` check above (L35). The parameter is an `int` therefore `None` can be expected to raise a `TypeError`. Then change this line to `DelaySeconds=delay_sec`.